packages/form-schema/src/widgetWrapper.tsx

```diff
@@ -45,6 +45,14 @@ const Wrapper = (Component, inputType: string = '') => {
       formProps.value = true;
     }
 
+    if (inputType === 'checkboxes') {
+      return (
+        <div>
+          <Component />
```

Comment:
I wasn't sure where to find the props that needed to be passed in here

Reply:
I think passing in <Component {...formProps} /> should mostly work, there may be some to add/remove